docs/output.md

```diff
@@ -279,3 +279,18 @@ These curves will be displayed in the pipeline run's MultiQC report, however you
 
 [DamageProfiler](https://github.com/Integrative-Transcriptomics/DamageProfiler)
 is a tool which calculates a variety of standard 'aDNA' metrics from a BAM file. The primary plots here are the misincorporation and length distribution plots. Ancient DNA undergoes depurination and hydrolysis, causing fragmentation of molecules into gradually shorter fragments, and cytosine to thymine deamination damage, that occur on the subsequent single-stranded overhangs at the ends of molecules.
+
+### Contamination estimation for Human DNA
+
+#### ANGSD nuclear contamination
+
+<details markdown="1">
+<summary>Output files</summary>
+
+- `contamination_estimation/angsd/`
+
+  - `*.txt`: Text file containing the results of nuclear contamination estimation with ANGSD.
```

Comment:
This should be updated to mention the print_contamination table as well as the individual library result files.